ShchyrbaOleksandr/HW7/task3.py

```diff
@@ -0,0 +1,8 @@
+def count_characters():
```

Comment:
good job
but look at the more optimized version of code

def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result